### PR TITLE
Change comments, add custom builds, onFinished, fix duration.

### PR DESCRIPTION
**Pull request by piecubed**

### CODE REVIEW

**Comment by WiRight** [comment]
Thank you @piecubed for your job!

I'll review your pull request soon

**Comment by piecubed** [comment]
Need to fix one thing with the custom duration, then its good to go.

**Comment by piecubed** [comment]
Checked everything, ready to merge.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-        setState(() {
-            // Such like hide timer and show other widget
-            _showTimer = false;
-        });
-    },
-),
+class MyWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Countdown(
```
Comment (WiRight) [inline README.md:12]: Add this refernece in `example` and will be done!

**Comment by piecubed** [comment]
Added it

**Comment by WiRight** [comment]
Good job! Thank you for your contribution

**Pull request merged**